EventFilter/Phase2TrackerRawToDigi/plugins/RawToClusterProducer.cc

```diff
@@ -0,0 +1,465 @@
+#include "FWCore/Framework/interface/Event.h"
```

Comment:
All .h files (or cc files if there is no .h file) should have a comment near the to explaining what they do.

Reply:
added a comment